app/models/organization.rb

```diff
@@ -141,16 +136,10 @@ def downcase_slug
     self.slug = slug&.downcase
   end
 
-  def update_articles
-    return unless saved_change_to_slug || saved_change_to_name || saved_change_to_profile_image
+  def conditionally_update_articles
+    return unless Article::ATTRIBUTES_CACHED_FOR_RELATED_ENTITY.detect { |attr| saved_change_to_attribute?(attr) }
 
-    articles.update(cached_organization: Articles::CachedEntity.from_object(self))
-  end
-
-  def update_articles_cached_organization
-    return unless saved_change_to_attribute?(:name)
-
-    articles.update(cached_organization: Articles::CachedEntity.from_object(self))
+    articles.each(&:save)
```

Comment:
this makes sense given the hook is now "before save". Saving will trigger the expected action.

also, the TODO comment earlier suggested a lot of the existing complication was related to pushing search documents into ES (and we don't do that any more).